identity-apps-core/apps/x509-certificate-authentication-portal/src/main/webapp/css/language-selector.css

```
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Let's indent the license URL to be consistent with the license header format in other files.

Fix the other places as well.

```suggestion
 *    http://www.apache.org/licenses/LICENSE-2.0
```

---

https://github.com/wso2/identity-apps/pull/6251/commits/cffb5e719c0c22d604cb852fb64167932b1e575a